Source/WebCore/Modules/webauthn/AuthenticatorSelectionCriteria.h

```diff
@@ -40,11 +40,16 @@ namespace WebCore {
 enum class AuthenticatorAttachment : uint8_t;
 
 struct AuthenticatorSelectionCriteria {
-    std::optional<AuthenticatorAttachment> authenticatorAttachment;
```

Comment:
Nit: Why are we changing the storage type, if we always access through a method that only returns an optional?